TypeCobol/Compiler/SqlAnalysis/SqlAbstractRuleChecker.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using TypeCobol.Analysis;
+
+namespace TypeCobol.Compiler.SqlAnalysis
+{
+    /// <summary>
+    /// This class is derivated from Qualis.CobolAnalyzers.RuleSystem.RuleChecker
+    /// Maybe we should think to put the basic RuleSystem in the Analysis namespace.
+    /// </summary>
+    internal abstract class SqlAbstractRuleChecker<T>
```

Comment:
To private analyzer project.

Reply:
Idem

Reply:
done in [7f2afb7](https://github.com/TypeCobolTeam/TypeCobol/pull/1928/commits/7f2afb778747d8f26ff0ce751781c2ab8845aa78)